### ISSUE TITLE
reward in new self critical

### ISSUE THREAD

**Issue by muaz1994** [ruotianluo/self-critical.pytorch#222]
Hi @ruotianluo and thanks for your work on this project.
In the implementation of the new self-critical, why are you printing the loss and not the reward:
out['struc_loss'] = struc_loss['loss']
out['reward'] = struc_loss['reward']
if struc_flag:
        print("iter {} (epoch {}), train_loss = {:.3f}, lm_loss = {:.3f}, struc_loss = {:.3f}, time/batch = {:.3f}" \
            .format(iteration, epoch, train_loss, model_out['lm_loss'].mean().item(), model_out['struc_loss'].mean().item(), end - start))

**Comment by ruotianluo** [comment]
I print the CIDER score out when computing the cider.
